DXMainClient/Online/IConnectionManager.cs

```diff
@@ -21,6 +22,8 @@ public interface IConnectionManager
 
         void OnWhoReplyReceived(string ident, string hostName, string userName, string extraInfo);
 
+        void OnWhoQueryComplete(string channel, List<Tuple<string, string, string, string>> whoList);
```

Comment:
Can you add a comment to explain the name `WhoQuery`? I feel it's hard to get the meaning from its name. Also, maybe a comment to explain what the 4 strings refers to in type `List<Tuple<string, string, string, string>>` , or, it's better to use `List<(string meaningfulArgName1, string meaningfulArgName2, string meaningfulArgName3, string meaningfulArgName4)>` if possible.